builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -253,6 +260,10 @@ func resourceAwsAutoscalingGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("vpc_zone_identifier", strings.Split(*g.VPCZoneIdentifier, ","))
 	d.Set("termination_policies", g.TerminationPolicies)
 
+	if g.EnabledMetrics != nil {
+		d.Set("enabled_metrics", flattenAsgEnabledMetrics(g.EnabledMetrics))
```

Comment:
Let's check the error on this, recently [bit by this][1]

[1]: https://github.com/hashicorp/terraform/pull/4779